public/app/features/explore/Explore.tsx

```diff
@@ -182,10 +183,42 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
     }
   };
 
+  /**
+   * Used by Logs details.
+   */
+  isFilterLabelActive = async (key: string, value: string) => {
+    if (this.props.queries.length === 0) {
+      return false;
+    }
+    for (const query of this.props.queries) {
+      const ds = await getDataSourceSrv().get(query.datasource);
+      if (!hasQueryManipulationSupport(ds, 'analyzeQuery') || !ds.analyzeQuery) {
+        return false;
+      }
+      const hasFilter = ds.analyzeQuery(query, {
```

Comment:
I am wondering, if in this case we need to know from which query is the log line coming so we correctly highlight filters. Because currently with 2 queries, it doesn't highlight anything.

Reply:
Yes, it's a known limitation that we should address as a follow up.

Reply:
Can you add comment about current limitation and that it does not work for more queries.

Reply:
Added. Let me know if you think this should be done in a different way.